aspnetcore/tutorials/nano-server.md

```diff
@@ -3,67 +3,71 @@ title: ASP.NET Core on Nano Server
+Nano Server is an installation option in Windows Server 2016, offering a tiny footprint, better security and better servicing than Server Core or full Server. Please consult the official [Nano Server documentation](https://technet.microsoft.com/en-us/library/mt126167.aspx) for more details and download links for 180 Days evaluation versions. 
+There are 3 easy ways for you to try out Nano Server, when you sign in with your MS account:
 
-1. You can download the Windows Server 2016 Technical Preview 5 ISO file, and build a Nano Server image
+1. You can download the Windows Server 2016 ISO file, and build a Nano Server image
 
-2. Download the Nano Server developer VHD
+2. Download the Nano Server VHD
 
 3. Create a VM in Azure using the Nano Server image in the Azure Gallery. If you don’t have an Azure account, you can get a free 30-day trial
 
-In this tutorial, we will be using the pre-built [Nano Server Developer VHD](https://msdn.microsoft.com/en-us/virtualization/windowscontainers/nano_eula)  from Windows Server Technical Preview 5.
+In this tutorial, we will be using the 2nd option, the pre-built Nano Server VHD from Windows Server 2016
 
 Before proceeding with this tutorial, you will need the [published](../publishing/index.md) output of an existing ASP.NET Core application. Ensure your application is built to run in a **64-bit** process.
 
+
 ## Setting up the Nano Server Instance
 
-[Create a new Virtual Machine using Hyper-V](https://technet.microsoft.com/en-us/library/hh846766.aspx) on your development machine using the previously downloaded VHD. The machine will require you to set an administrator password before logging on. At the VM console, press F11 to set the password before the first log in.
+[Create a new Virtual Machine using Hyper-V](https://technet.microsoft.com/en-us/library/hh846766.aspx) on your development machine using the previously downloaded VHD. The machine will require you to set an administrator password before logging on. At the VM console, press F11 to set the password before the first log in.  
+Then you also need to check your new VM's IP address either my checking your DHCP server, fixed IP supplied while provisioning your VM or in Nano Server recovery console's networking settings.
 
-After setting the local password, you will manage Nano Server using PowerShell remoting.
+> [!NOTE]
+> Let's assume your new VM runs with the local V4 IP address 192.168.1.10.
+
+Now you're able to manage it using PowerShell remoting, which is the only way to fully administer your Nano Server
 
 ### Connecting to your Nano Server Instance using PowerShell Remoting
 
 Open an elevated PowerShell window to add your remote Nano Server instance to your `TrustedHosts` list.
 
+> which is necessary when remoting with WinRM over HTTP (default port 5985). After importing a certificate and enabling 
```

Comment:
I wouldn't format this as blockquote.  Consider change "which" to "This" and putting the sentence directly after the code block.